crates/matrix-sdk/Cargo.toml

```diff
@@ -138,6 +138,7 @@ tokio = { workspace = true, features = ["fs", "rt", "macros"] }
 
 [dev-dependencies]
 anyhow = { workspace = true }
+assert-json-diff = "2.0"
```

Comment:
We are already using `assert-json-diff` inside `matrix-sdk-ui`. I think it's a good candidate for being declared inside the workspace, and use `{ workspace = true }` here.